app/src/main/java/org/simple/clinic/instantsearch/InstantSearchModel.kt

```diff
@@ -39,10 +39,11 @@ data class InstantSearchModel(
   companion object {
     fun create(
         additionalIdentifier: Identifier?,
-        patientPrefillInfo: PatientPrefillInfo?
+        patientPrefillInfo: PatientPrefillInfo?,
+        searchQuery: String?
```

Comment:
Is there a reason this change is part of this PR? 'Instant Search Improvements' is vague doesn't tell us what is actually happening in a particular changeset. Ideally, supporting the new way of pagination and support for prefilling search query should be two different PRs.

Reply:
I had to do this here mainly to avoid the load all patients method being called twice and causing issues with the paging source. Since we are not technically migrating from older paging implementation in this screen, I have decided to called it instant search screen improvements and make the necessary changes required for the instant search to work correctly. wdyt?

Reply:
We could make that change first and then switch to the new paging implementation, right? The two need not go together.

Reply:
Okay, let me break out these commits then.